Automattic/jetpack
# VideoPress: rename block to the `videopress/video` name

~rebased with https://github.com/Automattic/jetpack/pull/25849. Hold on...~

This PR applies several changes in the code based on the `videopress/video` block name. It means that it not only renames the block name but folders, methods, and calls.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress Pkg: apply changes according on the `videopress/video` name convention for the video block
* Update VideoPress and Jetpack plugins


#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Confirm the VideoPress block is properly registered (you'll get validation issues since it changes the block name)
* Check the item above in all environments: Local dev-env, Simple, Atomic and JN sites.

Testing on Simple sites

* Apply the patch in your sandbox to get the Fusioned changes
* Run the Jetpack downloader to apply the de-Fusiones changes
* Confirm VideoPress Video (v6) block is there. Probably you'll get some validation issues
* Confirm you can transform from/to current implement from/to v6

https://user-images.githubusercontent.com/77539/187229938-4fa5becc-1061-4cc4-83f0-5730af99a886.mov
